app/src/views/DrefFinalReportExport/index.tsx

```diff
@@ -710,83 +944,85 @@ export function Component() {
+                        className={styles.disabilitiesPopulation}
+                        label={strings.peopleWithDisabilitiesLabel}
+                        value={drefResponse?.disability_people_per}
+                        suffix="%"
                         valueType="number"
                         strongValue
                     />
-                )}
-                <BlockTextOutput
-                    className={styles.disabilitiesPopulation}
-                    label={strings.peopleWithDisabilitiesLabel}
-                    value={drefResponse?.disability_people_per}
-                    suffix="%"
-                    valueType="number"
-                    strongValue
-                />
-                {drefResponse?.type_of_dref !== DREF_TYPE_ASSESSMENT && (
+                    {drefResponse?.type_of_dref !== DREF_TYPE_ASSESSMENT && (
+                        <BlockTextOutput
+                            label={strings.boysLabel}
+                            value={drefResponse?.assisted_num_of_boys_under_18}
+                            valueType="number"
+                            strongValue
+                        />
+                    )}
+                    <div className={styles.emptyBlock} />
                     <BlockTextOutput
-                        label={strings.boysLabel}
-                        value={drefResponse?.assisted_num_of_boys_under_18}
+                        className={styles.metaItem}
+                        label={strings.totalAssistedPopulationLabel}
+                        value={drefResponse?.num_assisted}
                         valueType="number"
                         strongValue
                     />
-                )}
-                <div className={styles.emptyBlock} />
-                <BlockTextOutput
-                    className={styles.metaItem}
-                    label={strings.totalAssistedPopulationLabel}
-                    value={drefResponse?.num_assisted}
-                    valueType="number"
-                    strongValue
-                />
-                <div className={styles.emptyBlock} />
-                <BlockTextOutput
-                    label={strings.targetedPopulationLabel}
-                    value={drefResponse?.total_targeted_population}
-                    valueClassName={styles.totalTargetedPopulationValue}
-                    valueType="number"
-                    strongValue
-                />
-            </Container>
-            {showRiskAndSecuritySection && (
+                    <div className={styles.emptyBlock} />
+                    <BlockTextOutput
+                        label={strings.targetedPopulationLabel}
+                        value={drefResponse?.total_targeted_population}
+                        valueClassName={styles.totalTargetedPopulationValue}
+                        valueType="number"
+                        strongValue
+                    />
+                </Container>
+            )}
```

Comment:
@barshathakuri @frozenhelium , there are multiple conditions here and these seem a bit confusing. Can you guys double check? @barshathakuri can you paste here the requiement for this? I couldn't see it in the ticket.

Reply:
In the Imminent DREF Request Final Report file at the bottom of [this issue](https://github.com/IFRCGo/go-web-app/issues/1853), I’ve updated the PDF export to show or hide fields based on the export mockup.

I've uploaded the export mockup here.
[Imminent.DREF.request.Final.Report.docx](https://github.com/user-attachments/files/21248072/Imminent.DREF.request.Final.Report.docx)

Reply:
@AdityaKhatri Good catch. Uday has confirmed with the client that they do want this section included, so I'm updating the comment accordingly. Thank you!